src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -32,6 +33,20 @@
  */
 public class NumberUtils {
 
+    /** Reusable char constant for zero. */
+    public static final char CHAR_ZERO = '0';
```

Comment:
These are all superfluous. Named constants are for number, not string and char literals.